scripts/firebase-deploy

```diff
@@ -3,23 +3,25 @@
 main() {
 	set -euo pipefail
 	on_ci && set -x
-	local channel result url
+	local channel result url environ project
 
-	if (($# > 1)); then
-		exit_error "usage [CHANNEL=dev]"
+	if (($# > 2)); then
+		exit_error "usage [ENV=stage [CHANNEL=dev]]"
 	fi
 
 	on_ci && setup_ci_creds
 
-	channel=$(get_channel "${1:-}")
+	environ="${1:-stage}"
+	channel=$(get_channel "${2:-}")
+	project=$(jq -r ".projects.${environ}" firebase/.firebaserc)
 	if [[ "${channel}" == "live" ]]; then
-		firebase --config firebase/firebase.json deploy --only hosting
+		firebase --config firebase/firebase.json --project "${environ}" deploy --only hosting
```

Comment:
should this be `--project "${project}"` ?

Reply:
maybe yeah. I mean project is the firebase term, but environment is what it is to me / in my context.... I'll change it everywhere to project with a note on the first occurance or so.

Reply:
nope, `${environ}` is correct here.

From .firebaserc:
```
{
  "projects": {
    "default": "evy-lang-stage",
    "stage": "evy-lang-stage",
    "prod": "evy-lang"
  },
//...
}
```
here `stage`, `prod` are the `environ` and `evy-lang-stage` and `evy-lang` are the projects. In the given commands I need the environment. The project I only need to pick out the URL from the REST response.

Reply:
I have removed the project variable altogether to make it less confusing...